src/main/features/core/tray.js

```diff
@@ -180,5 +180,50 @@ Emitter.on('audiooutput:list', (event, devices) => {
   setContextMenu();
 });
 
+// Change the Icon if its playing
+Emitter.on('playback:isPlaying', () => {
+  if (process.platform === 'darwin') {
+    appIcon.setImage(path.resolve(`${__dirname}/../../../assets/img/playing/macTemplate.png`));
```

Comment:
These paths are always consistently different by only the file name (the path is the same across playstate changes).  We should extract these `if` statements to the top of this file and assign the file name to a constant.

This will remove code duplication